Resources/config/core.xml

```diff
@@ -66,7 +66,7 @@
         </service>
         <!-- twig -->
         <service id="sonata.admin.twig.global" class="Sonata\AdminBundle\Twig\GlobalVariables">
-            <argument type="service" id="service_container"/>
+            <argument type="service" id="sonata.admin.pool"/>
```

Comment:
I don't understand, can you please clarify in which cases there happen a BC break with this service definition?